package.json

```diff
@@ -23,13 +23,13 @@
     "url": "https://github.com/typescript-eslint/typescript-eslint/issues"
   },
   "scripts": {
-    "build": "npx nx run-many --target=build --parallel --exclude website --exclude website-eslint",
+    "build": "npx nx run-many -t build --parallel --exclude website --exclude website-eslint",
```

Comment:
> ```diff
> - --target
> + -t
> ```

I have a very mild preference for the explicit syntax, to make it more understandable to not-frequent-Nx users like me. Not a blocker at all from me though.

Reply:
I just think it’s much cleaner, happy to revert though